styles/scss/placeholders/_link.scss

```diff
@@ -0,0 +1,32 @@
+/// Placeholder that is used to reset browser default styles for anchor link.
+///
+/// ### Output
+/// ```scss
+/// .test {
+///   padding: 0;
+///   margin: 0;
+///   color: unset;
+///   text-decoration: none;
+///   &:hover {
+///     color: unset;
+///   }
+/// }
+/// ```
+///
+/// @access public
+/// @author Kristijan Maric
+/// @example
+/// .test {
+///   @extend %link-reset;
+/// }
+
+%list-reset {
```

Comment:
```suggestion
%link-reset {
```

Reply:
Also I believe the filename should be the same as the placeholder (so `link-reset.scss`).

Reply:
Not a bad idea, should I change it for all files that are `-reset`?